src/extract_wheels.py

```diff
@@ -4,6 +4,8 @@
 import subprocess
 import sys
 
+
+from . import namespace_pkgs
 from .wheel import Wheel
```

Comment:
These relative imports are a bit dodgy. Can we get rid of them?

Reply:
Yep, they are a remnant from old code. They can be replaced with `src.`